definitions/procedures/knowledge_base_article.rb

```diff
@@ -26,7 +26,8 @@ def kcs_documents
       'fix_cpdb_validate_failure' => 'https://access.redhat.com/solutions/3362821',
       'fix_db_migrate_failure_on_duplicate_roles' => 'https://access.redhat.com/solutions/3998941',
       'upgrade_puppet_guide_for_sat63' => 'https://access.redhat.com/documentation/en-us/red_hat_satellite/6.3/html/upgrading_and_updating_red_hat_satellite/upgrading_puppet-1',
-      'many_fact_values' => 'https://access.redhat.com/solutions/4163891'
+      'many_fact_values' => 'https://access.redhat.com/solutions/4163891',
+      'migrate_to_postgresql' => 'https://theforeman.org/manuals/1.24/index.html#3.7MigratingtoPostgreSQL'
```

Comment:
shouldn't we have separate KCS links for up and downstream? RH KCS are not available w/o an RH account, so it seems wrong to tell users "here is the doc for you (oh and you need to register)"

people might think we're trying to force them to become RH customers (which we dont)